portal-web/docroot/html/taglib/ui/organization_search_container_results/page.jsp

```diff
@@ -20,12 +20,14 @@
 OrganizationSearchTerms searchTerms = (OrganizationSearchTerms)request.getAttribute("liferay-ui:organization-search-container-results:searchTerms");
 LinkedHashMap<String, Object> organizationParams = (LinkedHashMap<String, Object>)request.getAttribute("liferay-ui:organization-search-container-results:organizationParams");
 long parentOrganizationId = GetterUtil.getLong(request.getAttribute("liferay-ui:organization-search-container-results:parentOrganizationId"));
+
+Indexer<?> indexer = IndexerRegistryUtil.nullSafeGetIndexer(Organization.class);
```

Comment:
Is there a performance penalty for using "nullSafeGetIndexer" versus "getIndexer" or other ways of retrieving the indexer? Since we're not actually indexing anything and just need to check if it's enabled, we should be getting it via the quickest way possible.